flink-python/pyflink/datastream/connectors/kafka.py

```diff
@@ -15,30 +15,24 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 ################################################################################
-import warnings
 from abc import ABC, abstractmethod
 from enum import Enum
-from typing import Dict, Union, List, Set, Callable, Any, Optional
+from typing import Dict, Union, Set, Callable, Any, Optional
 
 from py4j.java_gateway import JavaObject, get_java_class
-
-from pyflink.common import DeserializationSchema, TypeInformation, typeinfo, SerializationSchema, \
+from pyflink.common import DeserializationSchema, SerializationSchema, \
     Types, Row
 from pyflink.datastream.connectors import Source, Sink
 from pyflink.datastream.connectors.base import DeliveryGuarantee, SupportsPreprocessing, \
     StreamTransformer
-from pyflink.datastream.functions import SinkFunction, SourceFunction
 from pyflink.java_gateway import get_gateway
 from pyflink.util.java_utils import to_jarray, get_field, get_field_value
 
 __all__ = [
-    'FlinkKafkaConsumer',
-    'FlinkKafkaProducer',
     'KafkaSource',
     'KafkaSourceBuilder',
     'KafkaSink',
     'KafkaSinkBuilder',
-    'Semantic',
     'KafkaTopicPartition',
```

Comment:
Didn't we remove KafkaTopicPartition from Java?

Reply:
KafkaTopicPartition is corresponded to org.apache.kafka.common.TopicPartition class.
https://github.com/apache/flink-connector-kafka/blob/6f251eca9dc93cd88ff11e60d3ae321d3ac91e2e/flink-python/pyflink/datastream/connectors/kafka.py#L674